packages/cactus-common/src/main/typescript/key-converter.ts

```diff
@@ -0,0 +1,128 @@
+import KeyEncoder from "key-encoder";
+
+export enum KeyFormat {
+  Raw = "raw",
+  Hex = "hex",
+  PEM = "pem",
+}
+
+export class KeyConverter {
```

Comment:
I am little confused for the usage of this class. I 
If your intended usage was for browser, this class should be placed at tools or new one like "cactus-client-sdk".
Additionally, I might be wrong, but Typescript has standard API to handle cryptographic key data like, CryptoKey. https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.cryptokey.html
I thought we better to have a functionality to check whether key is valid or not.